src/wazuh_modules/vulnerability_scanner/src/vulnerabilityScannerFacade.cpp

```diff
@@ -214,95 +216,138 @@ void VulnerabilityScannerFacade::initWazuhDBEventSubscription()
 
 void VulnerabilityScannerFacade::vulnerabilityScanPolicyChange(Utils::RocksDBWrapper& stateDB) const
 {
-    const auto vulnerabilityDetectionEnabled = PolicyManager::instance().isVulnerabilityDetectionEnabled();
-
+    // Check if the vulnerability scanner was enabled/disabled
+    const std::string moduleState = PolicyManager::instance().isVulnerabilityDetectionEnabled() ? ENABLED : DISABLED;
```

Comment:
Aren't we adding overhead here using string, why do not simply use the returned boolean value?
The same review applies to all cases that involve ENABLED, DISABLED variables.

Reply:
The put methods for the rocksDB wrapper supports the following types:
- std::string
- rocksdb::Slice

The rocksdb::Slice can be constructed via:
- std::string
- std::string_view
- char*
- SliceParts